pkgs/development/python-modules/orjson/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib
+, buildPythonPackage
+, maturin
+, fetchFromGitHub
+, wheel
+, toml
+}:
+
+buildPythonPackage rec {
+  pname = "orjson";
+  version = "3.4.1";
+
+  src = fetchFromGitHub {
+    owner = "ijl";
+    repo = pname;
+    rev = version;
+    sha256 = "2+CZ4K4Sijt3ksjD6pVteRNNf+4z0iwZQon11b0+nyk=";
+  };
+
+  format = "pyproject";
+
+  nativeBuildInputs = [
+    maturin
+    wheel
+    toml
+  ];
+
+  meta = {
+    description = "Fast, correct Python JSON library supporting dataclasses, datetimes, and numpy";
+    license = with lib.licenses; [ 
+      asl20
+      mit
+    ];
```

Comment:
```suggestion
    license = with lib.licenses; [ asl20 mit ];
```

Reply:
I don't think that is an improvement. Separate lines for list items helps with merge conflicts. Granted, they're unlikely here.

Reply:
I don't think those licenses change anytime soon. Most meta blocks do this in one line.